### PR TITLE
Use intptr/uintptr to avoid warnings

### PR DESCRIPTION
**Pull request by ShFil119**
Not sure if 100% good usage, but mutes annoying warnings.

### CODE REVIEW

File: src/rw/VisibilityPlugins.cpp [hunk 1]
```diff
@@ -216,7 +216,7 @@ CVisibilityPlugins::RenderObjNormalAtomic(RpAtomic *atomic)
 }
 
 RpAtomic*
-CVisibilityPlugins::RenderAlphaAtomic(RpAtomic *atomic, int alpha)
+CVisibilityPlugins::RenderAlphaAtomic(RpAtomic *atomic, intptr alpha)
```
Comment (aap) [inline src/rw/VisibilityPlugins.cpp:219]: no, this is wrong.

File: src/vehicles/Automobile.cpp [hunk 2]
```diff
@@ -4558,7 +4558,7 @@ SetVehicleAtomicVisibilityCB(RwObject *object, void *data)
 }
 
 void
-CAutomobile::SetComponentVisibility(RwFrame *frame, uint32 flags)
+CAutomobile::SetComponentVisibility(RwFrame *frame, uintptr flags)
```
Comment (aap) [inline src/vehicles/Automobile.cpp:4561]: also wrong

File: src/vehicles/Automobile.h [hunk 3]
```diff
@@ -172,7 +172,7 @@ class CAutomobile : public CVehicle
 	void SetDoorDamage(int32 component, eDoors door, bool noFlyingComponents = false);
 
 	void Fix(void);
-	void SetComponentVisibility(RwFrame *frame, uint32 flags);
+	void SetComponentVisibility(RwFrame *frame, uintptr flags);
```
Comment (aap) [inline src/vehicles/Automobile.h:175]: also wrong

File: src/rw/VisibilityPlugins.h [hunk 4]
```diff
@@ -45,7 +45,7 @@ class CVisibilityPlugins
 
 	static RpAtomic *RenderWheelAtomicCB(RpAtomic *atomic);
 	static RpAtomic *RenderObjNormalAtomic(RpAtomic *atomic);
-	static RpAtomic *RenderAlphaAtomic(RpAtomic *atomic, int alpha);
+	static RpAtomic *RenderAlphaAtomic(RpAtomic *atomic, intptr alpha);
```
Comment (aap) [inline src/rw/VisibilityPlugins.h:48]: also wrong

File: src/rw/VisibilityPlugins.cpp [hunk 5]
```diff
@@ -236,7 +236,7 @@ CVisibilityPlugins::RenderFadingAtomic(RpAtomic *atomic, float camdist)
 {
 	RpAtomic *lodatm;
 	float fadefactor;
-	uint32 alpha;
+	uintptr alpha;
```
Comment (aap) [inline src/rw/VisibilityPlugins.cpp:239]: also wrong

**Comment by ShFil119** [comment]
closing for now

**Pull request closed without merging**